extensions/MozillaIAM/Extension.pm

```diff
@@ -192,6 +192,28 @@ sub userprefs_can_change_email_password {
   }
 }
 
+sub admin_editusers_action {
```

Comment:
Why can't this change be part of `object_end_of_set_all`?
In which workflows is `object_end_of_set_all()` called, and in which ones is this called instead?

Reply:
Unfortunately in editusers.cgi, legacy code still updates the group mapping tables using direct SQL and does not use the object methods properly like $user->set_all() etc. The REST API and other places do use the proper object methods for updating group permissions which is why we needed to update both places. I will look into fixing editusers.cgi properly in the future.